### DISCUSSION TITLE
how to require css without dot default ?

### DISCUSSION THREAD

**Discussion by whatwg6** [webpack/webpack#16022] [Q&A]
I was using css modules in my project, when upgrade to webpack@5 require css should with dot default. how to fix it ?

config

```code
{
  loader: 'css-loader',
  options: {
    importLoaders: 1,
    esModule: false,
    sourceMap: false,
    modules: {
      mode: 'local',
      localIdentName: IS_PRODUCTION ? '[hash:base64]' : '[path][name]__[local]',
    },
  },
},
```
reproduce
```code
// a.css
.app {
   width: 100px;
}
```

```code
import styles from "./a.css"
const styles1 = require('./a.css')
const styles2 = require('./a.css').default

console.log(styles) // { app: xxxx }
console.log(styles1) // {__esModule: true, default : xxx} 
console.log(styles2) // {app: xxx} 
```

unexpected
```code
const styles1 = require('./a.css')

console.log(styles1) // {__esModule: true, default : xxx} 
```

expected
```code
const styles1 = require('./a.css')

console.log(styles1) // {app: xxx}  
```

versions
- "webpack": "^5.73.0"
- "css-loader": "^6.7.1"

**Comment by vankop** [comment]
unsure maybe `esModule` should be `true`

**Reply by whatwg6** [reply]
esModule: true has similar behavior like {__esModule: true, default : xxx} ,the result was not expected!

**Reply by vankop** [reply]
@alexander-akait

**Reply by alexander-akait** [reply]
Not sure how dot related to the problem, if you set `esModule: true` (and it is `true` by default) you need to use `import styles from "./styles.css"` if you have `esModule: false` you can use `require`
